JumpscaleCore/tools/tester/TesterFactory.py

```diff
@@ -0,0 +1,78 @@
+from Jumpscale import j
+
+import os
+from unittest import TestCase, skip
+
+from parameterized import parameterized
+
+locations_libs = []
+locations_core = []
+locations_threebot = []
+
+
+class Tester:
+    def _get_all_tests(locations, path):
```

Comment:
better to move into a small python function, no need to have that as a staticmethod in a class even (why isn't it decorated with @staticmethod?)